engine/src/cmd/movable.cpp

```diff
@@ -5,9 +5,40 @@
 #include "unit_find.h"
 #include "star_system.h"
 #include "universe.h"
+#include "game_config.h"
 
 #include <iostream>
 
+static const float VELOCITY_MAX = GameConfig::GetVariable( "physics", "velocity_max", 10000);
+
+//for the heck of it.
+static const float humanwarprampuptime = GameConfig::GetVariable( "physics", "warprampuptime", 5);
```

Comment:
we should (possibly in another PR) make these variables more like `HUMAN_WARP_RAMP_UP_TIME` so they're more readable

Reply:
Let's do that in another PR. I'm actually still trying to work on #73 and this PR was just a way to solve an issue there.

Reply:
That's fine. It's part of a larger code cleanup. Honestly, I'd like to see a global instance of the Game config and then have classes load the config from the global instance and minimize the static vars, or rather use a different pattern of initialization:
- single static var used as a gate (initialized, defaults to false)
- if false, the first class loads the config values

Allows the vars to be defaulted in code, and then have their config loaded at run-time more controllably. Long term.